packages/web3/src/codec/either-codec.ts

```diff
@@ -15,28 +15,18 @@ GNU Lesser General Public License for more details.
 You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
-import { Parser } from 'binary-parser'
 import { Codec } from './codec'
+import { Reader } from './reader'
 
 export interface Either<L, R> {
   either: number
   value: L | R
 }
```

Comment:
Let's refactor this to use enum

`{ kind: Left, value: L } | { kind: Right, value: R }`